docs/user-guide.md

```diff
@@ -1,72 +1,47 @@
-# User guide for the ARROW API
+# Arrow.js - The correlation engine
 
+## 1. Content
 
-## 1. Introduction
+ 1. [Content](#Content)
```

Comment:
TOC Links funktionieren nicht, alles in () muss kleingeschrieben werden und genau mit der überschrift übereinstimmen. Leerzeichen mit - ersetzen